tests/src/CommonTestCase.php

```diff
@@ -45,8 +45,6 @@ protected function setupGLPIFramework() {
 
       $DB = new DB();
 
-      include_once (GLPI_ROOT . "/inc/timer.class.php");
```

Comment:
breaks tests, and should be useless as the autoloader is already enabled. I don't remember why this was added.